scripts/install/opt/seagate/cortx/motr/libexec/cluster_utility_function

```diff
@@ -0,0 +1,251 @@
+    run_cmd $node1 "sudo sed -i 's/^\(ExecStartPost\|ExecStopPost\)=\/usr\/sbin\/attrd_updater/#&/' \
+                        /usr/lib/systemd/system/hare-consul-agent-c1.service"
+    run_cmd $node2 "sudo sed -i 's/^\(ExecStartPost\|ExecStopPost\)=\/usr\/sbin\/attrd_updater/#&/' \
+                        /usr/lib/systemd/system/hare-consul-agent-c2.service"
+
+    run_cmd $node1 "sudo systemctl reset-failed"
+    run_cmd $node2 "sudo systemctl reset-failed"
+    run_cmd $node1 "sudo systemctl daemon-reload"
+    run_cmd $node2 "sudo systemctl daemon-reload"
+
+    echo 'Starting Consul agents..'
+    run_cmd $node1 "sudo systemctl start hare-consul-agent-c1"
+    run_cmd $node2 "sudo systemctl start hare-consul-agent-c2"
+
+    while [[ ($(get_leader) != $lnode) && ($(get_leader) != $rnode) ]]; do
+        sleep 1
+    done
+
+    # An issue is observed if we try to fetch data from Consul immediately
+    # after stating Consul services. So, let's wait till consul becomes available.
+    sleep 10
+    while ! [[ `/opt/seagate/cortx/hare/bin/consul kv get --recurse m0conf/nodes/srvnode-1 | grep confd` ]]
+    do
+        sleep 2
+    done
+
+    while ! [[ `/opt/seagate/cortx/hare/bin/consul kv get --recurse m0conf/nodes/srvnode-2 | grep confd` ]]
+    do
+        sleep 2
+    done
+    echo 'Started Consul agents..'
+}
+
+stop_consul() {
+    local node1=$lnode
+    local node2=$rnode
+    if $single_node; then
+        node1=$(/opt/seagate/cortx/hare/libexec/node-name)
+        node2=$(/opt/seagate/cortx/hare/libexec/node-name)
+    fi
+		
+    echo 'Stopping Consul agents..'
+    run_cmd $node2 "sudo systemctl stop hare-consul-agent-c2"
+    run_cmd $node1 "sudo systemctl stop hare-consul-agent-c1"
+
+    echo 'Deleting lnet interfaces..'
+    run_cmd $node1 "sudo lnetctl net del --net $net_type --if $liface:c1"
+    run_cmd $node2 "sudo lnetctl net del --net $net_type --if $riface:c2"
+
+    echo 'Restarting motr-kernel..'
+    run_cmd $node1 "sudo systemctl restart motr-kernel"
+    run_cmd $node2 "sudo systemctl restart motr-kernel"
+
+    echo 'Restoring Consul systemd units..'
+    run_cmd $node1 "sudo sed -i '/\(#ExecStartPost\|#ExecStopPost\)=\/usr\/sbin\/attrd_updater/s/#//g' \
+                        /usr/lib/systemd/system/hare-consul-agent-c1.service"
+    run_cmd $node2 "sudo sed -i '/\(#ExecStartPost\|#ExecStopPost\)=\/usr\/sbin\/attrd_updater/s/#//g' \
+                        /usr/lib/systemd/system/hare-consul-agent-c2.service"
+
+    run_cmd $node1 "sudo systemctl daemon-reload"
+    run_cmd $node2 "sudo systemctl daemon-reload"
+    run_cmd $node1 "sudo systemctl reset-failed"
+    run_cmd $node2 "sudo systemctl reset-failed"
+
+    echo 'Removing ipaddresses..'
+    run_cmd $node1 "ip addr del $ip1/24 dev $liface:c1"
+    run_cmd $node2 "ip addr del $ip2/24 dev $riface:c1"
+}
+
+
+
+if [[ $start_consul_only ]]; then
+    start_consul
+elif [[ $stop_consul_only ]]; then
+    stop_consul
+elif [[ $get_fid_only ]]; then
```

Comment:
I think this utility too should not be used, this will be  error-prone to maintain it.

Check if use this utility can be avoided.

Reply:
This is temporary. Once Hare/Privisioner can get the utility functions added to a file which can be accessed by everyone we will ditch this utility.

Reply:
Removed this file.